python_modules/libraries/dagster-aws/dagster_aws/utils/__init__.py

```diff
@@ -36,3 +40,11 @@ class ResourceWithBoto3Configuration(ConfigurableResource):
     profile_name: Optional[str] = Field(
         default=None, description="Specifies a profile to connect that session"
     )
+
+
+def ensure_dagster_aws_tests_import() -> None:
+    dagster_package_root = (Path(dagster_aws_init_py) / ".." / "..").resolve()
+    assert (
+        dagster_package_root / "dagster_aws_tests"
+    ).exists(), "Could not find dagster_aws_tests where expected"
+    sys.path.append(dagster_package_root.as_posix())
```

Comment:
if you forget to add this when adding a new _tests import, what's the experience you get today on BK? It would be sad if it works great until it tries to run on 3.12 in the release branch, then gets a confusing error

Reply:
the uv installs for the default 3.11 fail without this stuff so you should hit it on PR runs